src/scss/constants.scss

```diff
@@ -43,20 +43,58 @@ $playa-venue-live: rgb(125, 223, 194);
 $profile-image-bg-color: #999999;
 $room-info-bg: #005ee5;
 
+// @debt the following locations are using styled-components, so can't use our helper function yet
+//   pages/Admin/Room/Modal/RoomModal.styles.ts (z-index: 101)
+//   components/molecules/UserProfilePicture/UserProfilePicture.styles.ts (z-index: 10)
+//   components/atoms/Legend/Legend.styles.ts (z-index: 5)
+$z-layer-sidebar: 15;
 $z-layers: (
```

Comment:
[FYI] I didn't try to de-duplicate/perfectly name these values in this PR. In future I would like to move to more 'standard layers', but that is OOS of this PR. My aim here was mostly to have everything centralised and using our `z()` helper